src/test/java/uk/gov/ons/ctp/integration/fieldsvccucumber/cucSteps/sso/TestSSOFieldService.java

```diff
@@ -199,6 +144,18 @@ public void the_EQ_launch_event_is_triggered() {
     assertTrue(currentURL.contains("session") && currentURL.contains("token"));
   }
 
+  private String waitForEqLaunch() throws Exception {
+    String url = null;
+    for (int i = 0; i < 30; i++) {
+      url = driver.getCurrentUrl();
+      if (url != null && url.contains(testConfig.getEqHost())) {
+        break;
+      }
+      Thread.sleep(500);
```

Comment:
If you are editing this code I think that the sleep should be for a smaller amount of time (like say 5, 10, etc ms).
If it's going to poll it might as well not take an excess amount of time. I can only hope that small numbers would result in others also using small numbers elsewhere.

Reply:
done